go.mod

```diff
@@ -40,30 +40,40 @@ require (
 	gopkg.in/square/go-jose.v2 v2.6.0 // indirect
 )
 
+require (
```

Comment:
Is there any reason why this has been separated from the initial require block above?

Reply:
It looks like the direct and indirect imports were mixed up a long time a go, causing the package manager to get confused and try to create a 3rd require block. I've manually moved the new imports into the direct block.

Reply:
(manually moved them preserving case ordering)